angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead-selector/sprk-masthead-selector.component.ts

```diff
@@ -205,23 +184,6 @@ export class SprkMastheadSelectorComponent implements OnChanges {
    */
   @Input()
   analyticsString: string;
-  /**
-   * If `true`, the Masthead Selector will be open when rendered.
-   */
-  @Input()
-  isOpen = false;
-  /**
-   * The heading text displayed in
-   * in the box above the choices.
-   */
-  @Input()
-  heading: string;
```

Comment:
We deleted `heading` since it was only used in Dropdown, then we renamed `selector` to `heading` so the component is more readable.